react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,72 @@
+import AKButton from '@atlaskit/button';
+import React, { Component } from 'react';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog the represents a failure and allows a retry.
+ */
+class InlineDialogFailure extends Component {
```

Comment:
I'm wondering now if the naming should be changed. I think in the past you thought about ending components with the type of element the component represents. So this would be something like InlineFailureDialog, ending with Dialog to denote it's a dialog.